.pipelines/containerSourceData/scripts/BuildBaseContainers.sh

```diff
@@ -256,6 +280,13 @@ function docker_build_custom {
     local image_full_name=$2
     local final_image_to_use=$3
     local dockerfile=$4
+    local should_build=$5
+
+    if [[ $should_build != true ]]; then
```

Comment:
Same case here with YAML booleans.

In general, I think it might be easier and safer to catch this early, when we parse the input arguments. See `parse_yaml_boolean` inside `.pipelines/utilities/yaml_helpers.sh` in our pipelines repository. We could do something similar in here as well.

Reply:
updated